### PR TITLE
[zh-cn] add 2022-12-12-kubernetes-release-artifact-signing.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fixed: https://github.com/kubernetes/website/issues/40386

Translate Chinese version for 2022-12-12-kubernetes-release-artifact-signing.md

Preview：https://deploy-preview-40390--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2022/12/12/kubernetes-release-artifact-signing/

### CODE REVIEW

File: content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+上的公告）或参加我们的[周例会][meeting]。
+
+<!--
+We're looking forward to making even more of those awesome changes for future
+Kubernetes releases. For example, we're working on the [SLSA Level 3 Compliance
+in the Kubernetes Release Process][slsa] or the [Renaming of the kubernetes/kubernetes
+default branch name to `main`][kkmain].
+-->
+我们期待着在未来的 Kubernetes 版本中做出更多了不起地改变。例如，我们正在致力于 [Kubernetes
+发布过程中的 SLSA 3 级合规性][slsa]或将 [kubernetes/kubernetes 默认分支名称重命名为 `main`][kkmain]。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md:187]:
```suggestion
我们期待着在未来的 Kubernetes 版本中做出更多了不起的改变。例如，我们正在致力于
[Kubernetes 发布过程中的 SLSA 3 级合规性][slsa]或将 [kubernetes/kubernetes 默认分支名称重命名为 `main`][kkmain]。
```

File: content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+**作者：** Sascha Grunert
+
+**译者：** XiaoYang Zhang (HUAWEI)
+
+<!--
+The Kubernetes Special Interest Group (SIG) Release is proud to announce that we
+are digitally signing all release artifacts, and that this aspect of Kubernetes
+has now reached _beta_.
+-->
+Kubernetes 特别兴趣小组（SIG） Release 自豪地宣布，我们正在对所有发布工件进行数字签名，并且
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md:26]:
```suggestion
Kubernetes 特别兴趣小组 SIG Release 自豪地宣布，我们正在对所有发布工件进行数字签名，并且
```
感觉去掉括号更通顺一点，虽然跟 en 不一致了

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+**binary artifacts** to the signing process as well! This means that now all
+[client, server and source tarballs][tarballs], [binary artifacts][binaries],
+[Software Bills of Material (SBOMs)][sboms] as well as the [build
+provenance][provenance] will be signed using [cosign][cosign]. Technically
+speaking, we now ship additional `*.sig` (signature) and `*.cert` (certificate)
+files side by side to the artifacts for verifying their integrity.
+-->
+我们已经对所有官方发布的容器镜像进行了签名（从 Kubernetes v1.24 开始）。
+在 V1.24 版本和 V1.25 版本中，镜像签名是 alpha 版本。
+在 V1.26 版本中，我们将所有的 **二进制工件** 也加入到了签名过程中！
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md:56]:
```suggestion
在 v1.24 版本和 v1.25 版本中，镜像签名是 alpha 版本。
在 v1.26 版本中，我们将所有的 **二进制工件** 也加入到了签名过程中！
```
The version notation for Kubernetes should be consistent with the original text.

File: content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+-->
+我们期待着在未来的 Kubernetes 版本中做出更多了不起的改变。例如，我们正在致力于
+[Kubernetes 发布过程中的 SLSA 3 级合规性][slsa]或将 [kubernetes/kubernetes 默认分支名称重命名为 `main`][kkmain]。
+
+<!--
+Thank you for reading this blog post! I'd like to use this opportunity to give
+all involved SIG Release folks a special shout-out for shipping this feature in
+time!
+-->
+感谢您阅读这篇博文！我想借此机会向所有参与的 SIG Release 人员表示特别地感谢，感谢他们及时推出这一功能！
```
Comment (ydFu) [inline content/zh-cn/blog/_posts/2022-12-12-kubernetes-release-artifact-signing.md:194]:
```suggestion
感谢你阅读这篇博文！我想借此机会向所有参与的 SIG Release 人员表示特别地感谢，感谢他们及时推出这一功能！
```

**Review by ydFu — APPROVED** [review]
/lgtm

Thank you for localizing the blog post "Kubernetes 1.26: We're now signing our binary release artifacts!" into zh-cn.

**Pull request merged**